core_backend/app/llm_call/parse_input.py

```diff
@@ -116,6 +116,48 @@ async def wrapper(
     return wrapper
 
 
+def _process_identified_language_response(
+    identified_language: IdentifiedLanguage | None,
+    response: UserQueryResponse,
+    task: str = "LANGUAGE IDENTIFICATION",
+) -> UserQueryResponse | UserQueryResponseError:
+    """Process the identified language and return the response."""
+    supported_languages = IdentifiedLanguage.get_supported_languages()
+    supported_language_string = ", ".join(supported_languages)
+
+    match identified_language:
+        case IdentifiedLanguage.UNINTELLIGIBLE:
+            error_response = UserQueryResponseError(
+                error_message=STANDARD_FAILURE_MESSAGE
```

Comment:
Not sure about the inclusion of the `STANDARD_FAILURE_MESSAGE` here and in the other error cases. By default it says "Sorry, I am unable to find an answer to your question in the knowledge base." which is irrelevant here. Let's just drop that?

Reply:
The chat manager gets to decide what to show the user anyway - this is just extra info.

Reply:
I see that other guardrails are using the same thing. Added a note to myself to fix those in my next PR.